lib/matplotlib/backend_bases.py

```diff
@@ -3387,17 +3387,15 @@ def add_tools(self, tools):
         ----------
         tools : List
             List in the form
-            [[group1, [(Tool1, name1), (Tool2, name2) ...]][group2...]]
-            where group1 is the name of the group where the
-            Tool1, Tool2... are going to be added, and name1, name2... are the
-            names of the tools
+            [(Tool1, name1), (Tool2, name2) ...]
+            where Tool1, name1 represent the tool, and the respective name
+            of the tool which gets used as an id.
         """
 
-        for group, grouptools in tools:
-            for position, tool in enumerate(grouptools):
-                self.add_tool(tool[1], tool[0], group, position)
+        for tool, name in tools:
+            self.add_tool(name, tool)
 
-    def add_tool(self, name, tool, group=None, position=None):
+    def add_tool(self, name, tool, *args, **kwargs):
```

Comment:
Please remind me why we want to allow `*args` and `**kwargs` to be passed to the `tool.__init__`? this means, not all tools can be merged into master because the list of tools `backend_tools.tools` doesn't take arguments